#!/bin/sh /usr/share/dpatch/dpatch-run
## 331-magic-add-xcursor.dpatch by Mathias Brodala <info@noctus.net>
##
## DP: New magic for X11 cursor files (Closes: #451246).

@DPATCH@

diff -Naur file-4.21.orig/magic/Magdir/images file-4.21/magic/Magdir/images
--- file-4.21.orig/magic/Magdir/images	2007-12-23 17:40:53.000000000 +0000
+++ file-4.21/magic/Magdir/images	2007-12-23 17:40:46.000000000 +0000
@@ -534,3 +534,8 @@
 # URL:  http://www.jisyo.com/cgibin/view.cgi?EXT=pe5 (Japanese)
 # From: Simon Horman <horms@debian.org>
 0	string	IIO2H	Ulead Photo Explorer5
+
+# Type:	X11 cursor
+# URL:	http://webcvs.freedesktop.org/mime/shared-mime-info/freedesktop.org.xml.in?view=markup
+# From:	Mathias Brodala <info@noctus.net>
+0	string	Xcur	X11 cursor
